packages/bpk-docs/src/pages/IconsPage/IconsPage.js

```diff
@@ -58,6 +66,12 @@ const blurb = [
     Icons are provided in two sizes: small (18px) and large (24px). Both are
     pixel-snapped for clarity at the intended usage sizes.
   </Paragraph>,
+  <Paragraph>
+    <BpkButton href={`/${iconsSvgs}`}>
+      Download all icons
```

Comment:
Can we make it clear that they are downloading the svg source of the icons here? I also think making it a primary button makes it too inviting. As in engineers would download them too instead of using the react component. Thoughts?

Reply:
Nah that's a good point! I did a button because I felt that a link didn't really stand out, but maybe that's ok?

I've asked @willhg for his opinion but still waiting to hear.